src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -35,37 +38,64 @@
     private static final double motorSpeed = .6;
     private static final int motorStop = 0;
 
+    // Sets motors to inverted control mode.
     public Climber() {
         outsideClimberMotor1.setInverted(true);
         insideClimberMotor1.setInverted(true);
     }
 
+    // Initialize sets solenoids to false, or reverse, which makes it in the "Off" position
     public void initialize() {
```

Comment:
What does this function do? Are u thinking of the command `initialize()` function that u can override??

Reply:
Not sure, it's been here since Owen first made the subsystem. Shouldn't it be in the all of the climber's commands initialize?

Reply:
I'll remove then.